src/components/DeleteComment.jsx

```diff
@@ -0,0 +1,27 @@
+import { removeComment } from "../api";
+
+const user = "weegembump";
+
+function DeleteComments({ comment_id, setComments, author }) {
+  if (user === author) {
+    return (
+      <button
+        id="delete_btn"
+        onClick={() => {
+          removeComment(comment_id);
+          document.getElementById("delete_btn").disabled = true;
```

Comment:
try to avoid using dom manipulation with react unless you really need to, you could handle this by adding some kind of state :)